src/__tests__/index.test.ts

```diff
@@ -1,60 +1,92 @@
 import axios, { AxiosInstance } from "axios";
 import delay from "delay";
 import * as axiosCachingDns from "..";
-import { DNSEntryCache } from "../DNSEntryCache";
+import { Config } from "../index.d";
+import { clearStats } from "../axios-cached-dns-resolve";
 
 let axiosClient: AxiosInstance;
 let useRedis: boolean = false; // Flag to switch between Redis and LRU
 
 beforeEach(async () => {
-  axiosCachingDns.config.dnsTtlMs = 1000;
-  axiosCachingDns.config.dnsIdleTtlMs = 5000;
-  axiosCachingDns.config.cacheGraceExpireMultiplier = 2;
-  axiosCachingDns.config.backgroundScanMs = 100;
+  const testConfig = {
```

Comment:
We're using the `testConfig` custom config here instead of globally changing the config. This tests the custom config feature